compiler/noirc_evaluator/src/brillig/brillig_ir/entry_point.rs

```diff
@@ -9,7 +9,7 @@ use super::{
 };
 use acvm::acir::{brillig::MemoryAddress, AcirField};
 
-pub(crate) const MAX_STACK_SIZE: usize = 32768;
+pub(crate) const MAX_STACK_SIZE: usize = 2048;
```

Comment:
Accidentally committed this stack size increase when testing against the blob lib.